articles/quickstart/webapp/aspnet-core-beta/01-login.md

```diff
@@ -12,29 +12,31 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
+
 <!-- markdownlint-disable MD041 -->
 
-<%= include('../../../_includes/_new_app', { showClientSecret: false, isPublicClient: false }) %>
+<%= include('../../../\_includes/\_new_app', { showClientSecret: false, isPublicClient: false }) %>
 
 <!-- markdownlint-disable MD002 MD041 -->
+
 ### Configure Callback URLs
 
 The Callback URL of your application is the URL where Auth0 will redirect to after the user has authenticated in order for the SDK to complete the authentication process.
 
 You will need to add this URL to the list of Allowed URLs for your application in your [Application Settings](${manage_url}/#/applications), this URL will mostly take the format `https://YOUR_APPLICATION_URL/callback`.
 
-<%= include('../../../_includes/_logout_url', { returnTo: 'http://localhost:3000' }) %>
+<%= include('../../../\_includes/\_logout_url', { returnTo: 'http://localhost:3000' }) %>
```

Comment:
```suggestion
<%= include('../../../\_includes/\_logout\_url', { returnTo: 'http://localhost:3000' }) %>
```

Reply:
I reverted these changes, I had no intention to change anything regarding escaping underscores. This must have happened automatically.